src/main/resources/application-preprod.yaml

```diff
@@ -173,3 +192,4 @@ SKYGGE_SAK_URL: https://sak-q2.dev.intern.nav.no
 NORG2_URL: https://norg2.dev.adeo.no/norg2/
 REGOPPSLAG_URL: https://regoppslag.dev.intern.nav.no
 SFTP_HOST: b27apvl00364.preprod.local
+AXSYS_URL: https://axsys.dev.intern.nav.no/api
```

Comment:
Tipper integrasjonstestene feiler fordi du det ikke er oppdatert config i `application-integrasjonstest.yaml`.

kanskje vurdere å legge inn noe i de andre application.yaml-filene også for å kjøre lokalt/e2e (hvis denne brukes?)